src/models/Affiliation.js

```diff
@@ -0,0 +1,37 @@
+module.exports = function (sequelize, DataTypes) {
+  const Affiliation = sequelize.define(
+    'AffiliationModel',
+    {
+      cv_id: {
```

Comment:
En la BD se está usando el key `dni` como PK.

Reply:
sí en este caso, el id es diferente, lo voy a correjir

Reply:
solved in 
1417cfb